include/onnxruntime/core/session/environment.h

```diff
@@ -29,13 +32,20 @@ class Environment {
      Returns whether any runtime environment instance has been initialized.
   */
   static bool IsInitialized() { return is_initialized_; }
+  concurrency::ThreadPool* GetThreadPool() {
```

Comment:
Is there going to be a separate PR for this change?

Reply:
No. I forgot this file. Thanks.